meta/main.yml

```diff
@@ -27,27 +28,26 @@ galaxy_info:
   # this branch. If Travis integration is configured, only notifications for this
   # branch will be accepted. Otherwise, in all cases, the repo's default branch
   # (usually master) will be used.
-  #github_branch:
+  # github_branch:
 
   #
   # platforms is a list of platforms, and each platform has a name and a list of versions.
   #
   platforms:
-  - name: Amazon
-    versions:
-    - all
-  - name: EL
-    versions:
-    - all
+    - name: Amazon
+      versions:
+        - all
+    - name: EL
+      versions:
+        - all
 
   galaxy_tags: []
-    # List tags for your role here, one per line. A tag is a keyword that describes
-    # and categorizes the role. Users find roles by searching for tags. Be sure to
-    # remove the '[]' above, if you add tags to this list.
-    #
-    # NOTE: A tag is limited to a single word comprised of alphanumeric characters.
-    #       Maximum 20 tags per role.
-
-dependencies:
-    - mergermarket.pip
-    - mergermarket.docker
```

Comment:
We run the agent in Docker so we need Docker to be installed so we need our Ansible role for Docker, no?

Reply:
Just re-read your description on the pull request about the "new" syntax for dependencies. Does that install them just-in-time then?

Reply:
Going to move these back and let users of the role decide on how to manage dependencies